src/dispensary.js

```diff
@@ -142,26 +142,30 @@ export default class Dispensary {
     var files = [];
 
     for (let version of library.versions) {
-      if (library.filename) {
-        files.push({
-          file: library.filename,
-          fileOut: library.filenameOutput || library.filename,
-          index: index,
-          library: library,
-          version: version,
-          minified: false,
-        });
-      }
+      var minVersion = library.minVersion;
+      if (!minVersion || (minVersion && naturalCompare(
+          minVersion, version) <= 0)) {
```

Comment:
Can do. But naturalCompare behaves exactly like any _compare_ in terms of return values. The only differences is that it returns what you would expect from _compare("1.2.0", "1.11.0")